Create loginregister forms

HelChris opened the pull request:
Started js+css for:
- login form
- signup form
- terms of service (placeholder)
- forgot password (placeholder)
- forms.css +terms.css
- added font: Bebas Kai (tagline font)
- added img: Grim reaper
- added moon
- updated README.md

The css is not DRY, forms is not functional, but the skeleton of the forms is hopefully ready.

HelChris committed:
add files and folders for login and signup forms

HelChris committed:
add folders and files

HelChris committed:
update with changes from dev branch

HelChris committed:
add html to form

HelChris committed:
add font bebas kai

HelChris committed:
add grimreaper

HelChris committed:
add login and signup forms

HelChris committed:
update script link

HelChris committed:
update project structure

Knirkeli reviewed (changes requested):
Looks grate. Needs checks and warnings for wrong form input.

HelChris committed:
update forms with wrong form input error handeling and messages

HelChris committed:
add type=module

HelChris committed:
add text-wrap balance

HelChris committed:
add flex to .container to keep centered always

HelChris commented:
- updated with form and error handling for input fields. 
- cleaned up the css
- also allowed for norwegian characters in input

what do you think?

HelChris committed:
Merge branch 'dev' of https://github.com/Juniors-Dev/Frontend-Productivity-Graveyard into create-loginregister-forms

HelChris committed:
adjusted margin-top -220px bc of margin top inside header.css

HelChris committed:
remove decorative moon

HelChris committed:
remove margin top in terms

HelChris committed:
update meta tags in head

Knirkeli reviewed (approved):
lgtm :)
